index.html

```diff
@@ -400,6 +400,28 @@ <h3>
       <dfn class="export" data-dfn-for="DigitalCredential">[[\discovery]]</dfn>
       whose value is "remote".
     </p>
+    <h3>
+      `isProtocolSupportedByClient()` Method
+    </h3>
+    <p>
+      Digital credential verifiers and issuers use this static method to
+      determine which [=digital credential/exchange protocols=] the client
+      supports for interactions with credential providers.
+    </p>
+    <p>
+      This method does not convey protocol support in the underlying platform
+      or credential provider.
+    </p>
+    <p>
```

Comment:
```suggestion
    <p>
        User agents SHOULD NOT vary the response value based on any information about availability of hardware, presence or configuration of software, wallets or credentials, or user configuration or preferences. If the response value varied, the user agent would introduce risks both of fingerprinting and of silently revealing other details about user behavior or configuration. The response value SHOULD vary only by user agent major version and indicate whether the browser supports distributing requests with that protocol to underlying platform or provider.
    </p>
    <p>
```

Reply:
This is just a starting point proposed text for trying to capture the idea that this is just a convenience method indicating browser version support, rather than details to the site about what the user has configured.

Reply:
That sounds great @npdoty. thanks!